content/styles/04-templates/home.css

```diff
@@ -1,5 +1,11 @@
-.home {
+/* .home {
   font-size: 3.5rem;
   text-align: center;
   position: relative;
 }
+ */
+.article-list {
```

Comment:
see previous comment with `article-list`

Reply:
Adressed with b2f8501eb1e605b66ea5affc16dd5b7a0d098a54

Reply:
I meant [this](https://github.com/neomad-team/styleguide/blob/master/content/components.html#L362)